CodeModel/src/main/java/org/openzen/zenscript/codemodel/definition/ExpansionDefinition.java

```diff
@@ -25,6 +25,6 @@ public <C, R> R accept(C context, DefinitionVisitorWithContext<C, R> visitor) {
 
 	@Override
 	public String getName() {
-		return null;
+		return "(expansion of " + target + ")";
```

Comment:
💭 Question:  
These getName calls are only used for display purposes, right?
So this change should not have any behavioral change?